src/p6est.c

```diff
@@ -1636,7 +1636,7 @@ p6est_replace_column_join (p4est_t * p4est, p4est_topidx_t which_tree,
       }
     }
     if (replace_fn != NULL) {
-      replace_fn (p6est, which_tree, P4EST_CHILDREN, 1, outgoing, q, 1, 1,
+      replace_fn (p6est, which_tree, P4EST_CHILDREN, 4, outgoing, q, 1, 1,
```

Comment:
Good point, but having Toby confirm may still be the safest. Let's make this P4EST_CHILDREN for clarity.
Testing the change, does everything check out, i.e., the pointer to the outlayers contains 4 of them and they are valid?
While you're at it, if you have anything to add to the documentation of the replace function type please feel free.

Reply:
Done.